src/Ops.Web/Areas/Admin/Views/Features/Index.cshtml

```diff
@@ -0,0 +1,113 @@
+            <div class="modal-dialog" role="document">
+                <form asp-action="@nameof(FeaturesController.DeleteFeature)" method="post" role="form" style="display:inline">
+                    <input asp-for="Feature.Id" type="hidden" />
+                    <input asp-for="Feature.Name" type="hidden" />
+                    <div class="modal-content">
+                        <div class="modal-header">
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                            <h4 class="modal-title" id="deleteModalLabel">Delete Feature</h4>
+                        </div>
+                        <div class="modal-body">
+                            <span class="fa fa-exclamation-triangle" aria-hidden="true"></span>
+                            <span id="modal-text"></span>
+                        </div>
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-default" data-dismiss="modal" style="margin-right: 1em;">Cancel</button>
+                            <button type="submit" class="btn btn-danger btn-spinner pull-right" aria-label="Confirm">
+                                <span class="fas fa-spinner fa-times" aria-hidden="true"></span>
+                                Delete
+                                <span class="fa fa-spinner fa-pulse fa-lg fa-fw hidden"></span>
+                            </button>
+                        </div>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+
+    <div class="row" style="padding: 10px 0;">
+        <div class="col-xs-12">
+            <a asp-action="@nameof(FeaturesController.AddFeature)" class="btn btn-primary">Add a Feature</a>
+        </div>
+    </div>
+    <div class="row" style="padding: 10px 0;">
+        <div class="col-xs-12">
+            @if (Model.AllFeatures.Count() == 0)
+            {
+                <div class="alert alert-warning">No Features found.</div>
+            }
+            else
+            {
+                <div>
+                    <table class="table table-condensed table-bordered link-table">
+                        <thead>
+                            <tr>
+                                <th>Name</th>
+                                <th>Stub</th>
+                                <th>BodyText</th>
+                                <th>&nbsp;</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @foreach (var feature in Model.AllFeatures)
+                            {
+                                <tr>
+                                    <td class="td-class">
+                                        <a asp-action="@nameof(FeaturesController.Feature)" asp-route-featureName="@feature.Name.ToLower().Replace(" ", "")" class="rowlink">
```

Comment:
Why are we doing a `.Replace` for spaces all over the place? Are they being stored in the database?